src/path/index.js

```diff
@@ -20,9 +19,7 @@ const getTarget = (doc, path) => {
     }
     if (!result) {
       throw new TypeError(
-        `path ${path.toString()} does not match doc ${JSON.stringify(
-          toSlateDoc(doc)
-        )}`
+        `path ${path.toString()} does not match doc ${JSON.stringify(doc)}`
```

Comment:
How sold are you on not using `toSlateDoc` here? I believe that `doc` is a Yjs representation here, and in my experience, JSON-stringify-ing Yjs representations doesn't yield terribly useful output.

Reply:
I wanted to keep it, but the doc variable that is passed to the getTarget is not the top level doc that toSlateDoc expects to receive. The doc here is just the array of nodes for the document. I tried wrapping in a doc but thought it was too hack-y.